File: lib/backend/models/donate_model.js
const mongoose = require('mongoose');

const donateScheme = mongoose.Schema(
    {
        uid: {
            type: String,
            required: true,
        },
        title: {
            type: String,
            required: true,
        },
        des: {
            type: String,
            required: true,
        },
        time: {
            type: String,
            required: true,
        },
        urls: [
                {
                    type: String,
                    require: true,
                },
        ],
        address: {
            type: String,
            required: true,
        },
        latlng: {
            latitude: {
                type: Number,
            },
            longitude: {
                type: Number,
            },
        }
    }
);

const schema = mongoose.model('donateScheme', donateScheme);

module.exports = schema;

File: lib/models/donate_model.dart
import 'package:latlong2/latlong.dart';

class DonateModel{
  String title, des, address, uid, time;
  LatLng? latlng;  List<dynamic> urls;


  DonateModel({required this.title, required this.address, required this.latlng, required this.des, required this.uid, required this.time, required this.urls});
}